src/providers/v4/subgraph-provider.ts

```diff
@@ -21,6 +23,8 @@ export interface V4SubgraphPool {
 export type V4RawSubgraphPool = {
   id: string;
   feeTier: string;
+  tickSpacing: string; // TODO: waiting on v4 subgraph to return tickSpacing
+  hooks: string;
```

Comment:
curious how this `hooks` value will look like?

Reply:
aw hooks address will be mined prior to its deployment along with the pool. Uniswap Foundation has the [HookMiner](https://github.com/uniswapfoundation/v4-template/blob/b47e8f622852ea4bebbc0afb131d0c9458ecb8a7/test/utils/HookMiner.sol#L4) implementation. Hook devs don't have to use this HookMiner, but the `hooks` will follow certain pattern.